bundle/language_en.properties

```
@@ -662,6 +664,52 @@ angal.hospital.visitduration.txt
 angal.hospital.visitendhour.txt                                                                        = Visit end hour
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
 angal.lab.allnegative.txt                                                                              = All negative.
+angal.inventory.acanceledinventorycannotbemodified.msg                                                 = A canceled inventory can not be modified
```

```suggestion
angal.inventory.acanceledinventorycannotbemodified.msg                                                 = A canceled inventory cannot be modified.
```
In this diff `can not` is used twice and `cannot` is used 10 times.

Also messages should be statements that end in a period.